lib/drivers/node-mongodb-native/collection.js

```diff
@@ -168,7 +168,9 @@ function iter(i) {
       } else if (debug instanceof stream.Writable) {
         this.$printToStream(_this.name, i, args, debug);
       } else {
-        this.$print(_this.name, i, args, typeof debug.color === 'undefined' ? true : debug.color);
+        const color = typeof debug.color === 'undefined' ? true : debug.color;
```

Comment:
Really good job on this PR!

However, one minor change I'd do would be to change `typeof debug.color === 'undefined'` to `debug.color == null` which is the equivalent of `debug.color === null || debug.color === undefined` to also check for `null`.